Installed npm resolving to incorrect path

Installing a custom npm version as default isn't working for me (nor is pinning it). 



Running volta 0.8.2 on Windows 10 Build 17763.

```
λ volta install npm@6.13.4
success: installed and set npm@6.13.4 as default
```

```
λ volta list all
⚡⚡️ User toolchain:

    Node runtimes:
        v12.15.0 (default)

    Package managers:
        npm:
            v6.13.4 (default)

    Packages:
```

```
λ npm -v
internal/modules/cjs/loader.js:796
    throw err;
    ^

Error: Cannot find module 'C:\Users\me\AppData\Local\Volta\tools\image\npm\6.13.4\bin\node_modules\npm\bin\npm-cli.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:793:17)
    at Function.Module._load (internal/modules/cjs/loader.js:686:27)
    at Function.Module.runMain (internal/modules/cjs/loader.js:1043:10)
    at internal/main/run_main_module.js:17:11 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
internal/modules/cjs/loader.js:796
    throw err;
    ^

Error: Cannot find module 'C:\Users\me\AppData\Local\Volta\tools\image\npm\6.13.4\bin\node_modules\npm\bin\npm-cli.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:793:17)
    at Function.Module._load (internal/modules/cjs/loader.js:686:27)
    at Function.Module.runMain (internal/modules/cjs/loader.js:1043:10)
    at internal/main/run_main_module.js:17:11 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
```

The path it's trying to use doesn't match what is actually within my Volta AppData dir:

```
λ ls 'C:\Users\me\AppData\Local\Volta\tools\image\npm\6.13.4\bin'
node-gyp-bin  npm  npm-cli.js  npm.cmd  npx  npx-cli.js  npx.cmd```
```

also
```
λ volta uninstall npm@6.13.4
warning:  No package 'npm@6.13.4' found to uninstall

λ volta uninstall npm
error: Uninstalling npm is not supported yet.
```

If I just install node and not npm, npm resolves correctly to the version bundled with node (which happens to also be 6.13.4 in this case).

--- a/crates/volta-core/src/tool/npm/fetch.rs
+++ b/crates/volta-core/src/tool/npm/fetch.rs
@@ -175,7 +175,7 @@ node "$basedir/{}-cli.js" "$@"
 #[cfg(windows)]
 fn overwrite_cmd_launcher(base_path: &Path, tool: &str) -> Fallible<()> {
     write(
-        base_path.join(tool),
+        base_path.join(format!("{}.cmd", tool)),
         // Note: Adapted from the existing npm/npx cmd launcher, without unnecessary detection of Node location
         format!(
             r#"@ECHO OFF

